odxtools/dtcdop.py

```diff
@@ -130,22 +130,35 @@ def decode_from_pdu(self, decode_state: DecodeState) -> ParameterValue:
             sdgs=[],
         )
 
-    @override
-    def encode_into_pdu(self, physical_value: Optional[ParameterValue],
-                        encode_state: EncodeState) -> None:
-        if isinstance(physical_value, DiagnosticTroubleCode):
-            trouble_code = physical_value.trouble_code
-        elif isinstance(physical_value, int):
+    def numerical_trouble_code(self, dtc_value: ParameterValue) -> int:
```

Comment:
function names usually start with a verb

Reply:
do you prefer `get_numerical_trouble_code()`, `convert_to_numerical_trouble_code()` or something else?

Reply:
Either would be fine

Reply:
okay, I renamed it to [`convert_to_numerical_trouble_code()`](https://github.com/mercedes-benz/odxtools/pull/321/commits/4e79723736bb6d233bbe850cb439860282ead605)